### PR TITLE
add chinese Log/Information

### PR DESCRIPTION
**Pull request by kongslove**
All Chinese logs/information are supplemented with English logs/information

### CODE REVIEW

**Comment by kongslove** [comment]
All Chinese logs/information are supplemented with English logs/information

File: linkis-public-enhancements/linkis-bml/linkis-bml-client/src/main/scala/com/webank/wedatasphere/linkis/bml/client/impl/HttpBmlClient.scala [hunk 1]
```diff
@@ -36,47 +37,28 @@ import org.apache.commons.io.IOUtils
...
+  val clientConfig:ClientConfig = ClientConfigBuilder.newBuilder().addServerUrl(serverUrl)
+    .connectionTimeout(5 * 60 * 1000).discoveryEnabled(false)
+    .loadbalancerEnabled(false).maxConnectionSize(maxConnection)
+    .retryEnabled(false).readTimeout(readTimeout)
+    .setAuthenticationStrategy(authenticationStrategy).setAuthTokenKey(BmlConfiguration.AUTH_TOKEN_KEY.getValue)
+    .setAuthTokenValue(BmlConfiguration.AUTH_TOKEN_VALUE.getValue).build()
+  val dwsClientConfig:DWSClientConfig = new DWSClientConfig(clientConfig)
+  dwsClientConfig.setDWSVersion(Configuration.LINKIS_WEB_VERSION.getValue)
+  val dwsClientName:String = "BML-Client"
+  val dwsClient:DWSHttpClient = new DWSHttpClient(dwsClientConfig, dwsClientName)
```
Comment (wushengyeyouya) [inline linkis-public-enhancements/linkis-bml/linkis-bml-client/src/main/scala/com/webank/wedatasphere/linkis/bml/client/impl/HttpBmlClient.scala:57]: This file is not the latest version, please delete it.

File: linkis-public-enhancements/linkis-datasource/linkis-metadata/src/main/java/com/webank/wedatasphere/linkis/metadata/service/impl/MdqServiceImpl.java [hunk 2]
```diff
@@ -288,6 +289,20 @@ public static void main(String[] args) {
...
+        //partitions.stream().forEach();
+        List<MdqTablePartitionStatisticInfoVO> list = new ArrayList<>();
+        FileStatus tableFile = getRootHdfs().getFileStatus(new Path(tableLocation));
+        FileStatus[] fileStatuses = getRootHdfs().listStatus(tableFile.getPath());
+        for (FileStatus fileStatuse : fileStatuses) {
+            list.add(create(fileStatuse.getPath().toString()));
+        }
+        return list;
+    }*/
+
```
Comment (wushengyeyouya) [inline linkis-public-enhancements/linkis-datasource/linkis-metadata/src/main/java/com/webank/wedatasphere/linkis/metadata/service/impl/MdqServiceImpl.java:305]: Please delete it.

File: linkis-commons/linkis-protocol/src/main/scala/com/webank/wedatasphere/linkis/protocol/utils/ZuulEntranceUtils.scala [hunk 3]
```diff
@@ -49,6 +49,12 @@ object ZuulEntranceUtils {
       val shortExecID = longExecID.substring(6 + creatorLength + executeLength + instanceLength, longExecID.length)
       Array(creator, executeApplicationName, instance, shortExecID)
     }
+    /*val executeLength = Integer.parseInt(longExecID.substring(0,2))
+    val instanceLength = Integer.parseInt(longExecID.substring(2,4))
+    val executeApplicationName:String = longExecID.substring(4, 4 + executeLength)
+    val instance:String = longExecID.substring(4 + executeLength, 4 + executeLength + instanceLength)
+    val shortExecID:String = longExecID.substring(4 + executeLength + instanceLength, longExecID.length)
+    Array[String](executeApplicationName, instance, shortExecID)*/
```
Comment (wushengyeyouya) [inline linkis-commons/linkis-protocol/src/main/scala/com/webank/wedatasphere/linkis/protocol/utils/ZuulEntranceUtils.scala:57]: Please delete it.

File: linkis-commons/linkis-protocol/src/main/scala/com/webank/wedatasphere/linkis/protocol/utils/ZuulEntranceUtils.scala [hunk 4]
```diff
@@ -120,14 +126,20 @@ object ZuulEntranceUtils {
...
     val instanceStrLength = getStrFixedLen(instanceStr, SPLIT_LEN)
     EXEC_ID + applicationNameLength + instanceStrLength + applicationName + instanceStr + shortExecID
   }
 
+
+  /*private def getLengthStr(string:String):String = {
+    val length = string.length
+    if (length >= 10) String.valueOf(length) else "0" + String.valueOf(length)
+  }*/
+
```
Comment (wushengyeyouya) [inline linkis-commons/linkis-protocol/src/main/scala/com/webank/wedatasphere/linkis/protocol/utils/ZuulEntranceUtils.scala:142]: Please delete it.

File: linkis-commons/linkis-storage/src/main/scala/com/webank/wedatasphere/linkis/storage/domain/Dolphin.scala [hunk 5]
```diff
@@ -23,8 +23,8 @@ import com.webank.wedatasphere.linkis.storage.exception.StorageWarnException
 import com.webank.wedatasphere.linkis.storage.utils.{StorageConfiguration, StorageUtils}
 
 /**
-  * Created by johnnwang on 10/17/18.
-  */
+ * Created by johnnwang on 10/17/18.
```
Comment (wushengyeyouya) [inline linkis-commons/linkis-storage/src/main/scala/com/webank/wedatasphere/linkis/storage/domain/Dolphin.scala:26]: Please delete author information.

File: linkis-computation-governance/linkis-entrance/src/main/java/com/webank/wedatasphere/linkis/entrance/restful/EntranceRestfulApi.java [hunk 6]
```diff
@@ -257,18 +257,18 @@ public Response log(@Context HttpServletRequest req, @PathParam("id") String id)
...
             } catch (IllegalStateException e) {
-                logger.error("为 {} 获取日志失败 原因：{}", job.get().getId(), e.getMessage());
+                logger.error("Failed to get log information for :{}(为 {} 获取日志失败)", job.get().getId(), job.get().getId(), e.getMessage());
                 message = Message.ok();
                 message.setMethod("/api/entrance/" + id + "/log");
                 message.data("log", "").data("execID", id).data("fromLine", retFromLine + fromLine);
             } catch (final IllegalArgumentException e) {
-                logger.error("为 {} 获取日志失败", job.get().getId());
+                logger.error("Failed to get log information for :{}(为 {} 获取日志失败)", job.get().getId(), job.get().getId());
                 message = Message.ok();
```
Comment (wushengyeyouya) [inline linkis-computation-governance/linkis-entrance/src/main/java/com/webank/wedatasphere/linkis/entrance/restful/EntranceRestfulApi.java:266]: The original code here has BUGs,  the return message should use Message.error. Please help optimize it.

File: linkis-computation-governance/linkis-entrance/src/main/java/com/webank/wedatasphere/linkis/entrance/restful/EntranceRestfulApi.java [hunk 7]
```diff
@@ -257,18 +257,18 @@ public Response log(@Context HttpServletRequest req, @PathParam("id") String id)
...
                 }
             } catch (IllegalStateException e) {
-                logger.error("为 {} 获取日志失败 原因：{}", job.get().getId(), e.getMessage());
+                logger.error("Failed to get log information for :{}(为 {} 获取日志失败)", job.get().getId(), job.get().getId(), e.getMessage());
                 message = Message.ok();
                 message.setMethod("/api/entrance/" + id + "/log");
                 message.data("log", "").data("execID", id).data("fromLine", retFromLine + fromLine);
             } catch (final IllegalArgumentException e) {
-                logger.error("为 {} 获取日志失败", job.get().getId());
+                logger.error("Failed to get log information for :{}(为 {} 获取日志失败)", job.get().getId(), job.get().getId());
```
Comment (wushengyeyouya) [inline linkis-computation-governance/linkis-entrance/src/main/java/com/webank/wedatasphere/linkis/entrance/restful/EntranceRestfulApi.java:265]: Please use logger print exception info.

File: linkis-computation-governance/linkis-entrance/src/main/java/com/webank/wedatasphere/linkis/entrance/restful/EntranceRestfulApi.java [hunk 8]
```diff
@@ -257,18 +257,18 @@ public Response log(@Context HttpServletRequest req, @PathParam("id") String id)
                     retLog = sb.toString();
                 }
             } catch (IllegalStateException e) {
-                logger.error("为 {} 获取日志失败 原因：{}", job.get().getId(), e.getMessage());
+                logger.error("Failed to get log information for :{}(为 {} 获取日志失败)", job.get().getId(), job.get().getId(), e.getMessage());
```
Comment (wushengyeyouya) [inline linkis-computation-governance/linkis-entrance/src/main/java/com/webank/wedatasphere/linkis/entrance/restful/EntranceRestfulApi.java:260]: The stack trace will be lost.

**Review by wushengyeyouya — APPROVED** [review]
LGTM.

**Pull request closed without merging**